app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/NiaApp.kt

```diff
@@ -164,6 +164,7 @@ fun NiaApp(appState: NiaAppState) {
                     Column(Modifier.fillMaxSize()) {
                         // Show the top app bar on top level destinations.
                         val destination = appState.currentTopLevelDestination
+                        val shouldShowTopAppBar = destination != null
```

Comment:
How about managing this code like `shouldShowBottomBar` in the `NiaAppState` class

Reply:
I thought about that, but it makes the logic a little bit tricky here since we would still need to check that `destination` is not null below.

Reply:
Its Null=$sgsike2914 my apologies its early